File: Codechef/Practice/Alice and Marks.c
https://www.codechef.com/practice/course/c/PCL03/problems/MARKSTW?tab=statement




#include <stdio.h>

int main() {
    int x, y;
    scanf("%d %d", &x, &y);
    // Your code goes here (if needed)
    if(x >= 2*y)
        printf("Yes");
    else
        printf("No");
}

File: Codechef/Practice/Alice and Marks.go
https://www.codechef.com/practice/course/go/PGOL03/problems/MARKSTW?tab=statement




package main
import "fmt"

func main(){
    var alice,bob int
    fmt.Scan(&alice,&bob)
    if(alice>=2*bob){
        fmt.Println("YES")
    }else{
        fmt.Println("NO")
    }
    
}